### PR TITLE
add symbol to market data transform

### PR DESCRIPTION
**Pull request by andygnewman**
cc: @i-like-robots 

Fixes this issue https://github.com/Financial-Times/next-stream-page/issues/1018

### CODE REVIEW

File: server/transforms/markets-data.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ function formatMarketsData(item) {
 		timeStamp: item.quote.timeStamp,
 		volumeTraded: formatVolume(item.quote.volume),
 		changePercent: item.quote.change1DayPercent.toFixed(2),
-		changeDirection: changeDirection(item.quote.change1Day)
+		changeDirection: changeDirection(item.quote.change1Day),
+		symbol
```
Comment (bjfletcher) [inline server/transforms/markets-data.js]: Text editor fail? Shouldn't this `symbol` line be indented to indicate that it's a continuation of the previous line?

Comment (andygnewman) [inline server/transforms/markets-data.js]: I don't think so. It's shorthand for symbol: symbol as a new key: value pair to be returned.

Comment (i-like-robots) [inline server/transforms/markets-data.js]: I don't think we need to pass the `symbol` around, it's in the data (`item.basic.symbol`) =]

**Comment by i-like-robots** [comment]
🚢

**Pull request merged**